### DISCUSSION TITLE
Question about WBF

### DISCUSSION THREAD

**Discussion by IAmNewToCode** [ZFTurbo/Weighted-Boxes-Fusion#52] [Q&A]
Hi! I would just like to ask:
If I have 2 bbox coordinates generated by 2 different models in this format
[
  [
       [x1,y1,w,h], # for model 1
       [x2,y2,w,h],
  ],
 [
       [x2,y2,w,h], #for model 2 where the orders are flipped so x2 y2 is infront now
       [x1,y1, w, h] #for model 2
]]
will the module be able to determine that it is supposed to compare between the x1s instead of comparing index 0 of first list with index 0 of second list?

**Comment by ZFTurbo** [comment]
It's ok if order of boxes is different for different models.
